db/migrate/20231018165212_create_users.rb

```diff
@@ -0,0 +1,10 @@
+class CreateUsers < ActiveRecord::Migration[7.1]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.text :bio
+      t.timestamps
```

Comment:
- [ ] As stated in the project requirement Kindly make sure that Table and column names should match the ERD diagram, But your migration for creating users table miss the `posts_counter: `An integer column to store the count of posts by the user. So kindly add this column and run the migration again , it should look likes this 
  ```
create_table :users do |t|
    t.string :name
    t.string :photo # as photo should be a link to an image not a file
    t.text :bio
    t.integer :posts_counter
    t.timestamps # this will crate updated_at and created_at columns
  end

```